modules/features/settings/src/main/res/layout/fragment_settings_appearance.xml

```diff
@@ -20,6 +20,7 @@
     </com.google.android.material.appbar.AppBarLayout>
 
     <androidx.core.widget.NestedScrollView
+        android:id="@+id/nestedScrollView"
```

Comment:
Added to keep the scroll position after coming back to the screen from advanced settings.